### ISSUE TITLE
Docs for Spock-api-ghcjs missing

### ISSUE THREAD

**Issue by Profpatsch** [agrafix/Spock#93]
_(Original title: `docs for Spock-server-api missing`)_

Hackage isn’t building and there are no docs on the AWS instance (like for `Spock` and `Spock-api`).

**Comment by agrafix** [comment]
It's all there?

https://spockdocs.s3.eu-central-1.amazonaws.com/Spock-0.11.0.0/Web-Spock.html

https://spockdocs.s3.eu-central-1.amazonaws.com/Spock-core-0.11.0.0/index.html

https://spockdocs.s3.eu-central-1.amazonaws.com/Spock-api-0.11.0.0/index.html

**Comment by Profpatsch** [comment]
Ah, I meant the docs for `Spock-server-api`. Wrong formulation.

**Comment by agrafix** [comment]
It's right there: https://spockdocs.s3.eu-central-1.amazonaws.com/Spock-api-server-0.11.0.0/index.html ;-)

**Comment by agrafix** [comment]
Here's the whole ecosystem: https://spockdocs.s3.eu-central-1.amazonaws.com/index.html

**Comment by agrafix** [comment]
Only the docs for `Spock-api-ghcjs` are missing, which is essentially:

https://github.com/agrafix/Spock/blob/0.11.0.0/Spock-api-ghcjs/src/Web/Spock/Api/Client.hs#L27-L40

**Comment by Profpatsch** [comment]
Oh, I see.

**Comment by Profpatsch** [comment]
Maybe you could give `docs.spock.li` a DNS entry to the AWS server.

**Comment by agrafix** [comment]
Yeah, the problem is that HTTPS won't work anymore with plain `S3` if you use custom domains. The options are to configure `CloudFront` in top of `S3` or just a plain simple redirection.

**Comment by agrafix** [comment]
Maybe another way to approach this would be to get the Hackage-Docs working again. This is probably currently timing out because the package constraints are not "good" enough...
